From bc5f4887aeddd90a543300589e549d2ed390e1f7 Mon Sep 17 00:00:00 2001
From: Neal Liu <neal_liu@aspeedtech.com>
Date: Thu, 11 Sep 2025 14:19:42 +0800
Subject: [PATCH 22/32] configs: evb-ast2700: disable FIT signature configure
 temporarily

Disable FIT signature configure temporarily.

Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
Change-Id: Ief6330c666281238815073f70773dfded258d0ee
(cherry picked from commit 79d0b7627412e5506890bcd6c943a26fa2c0c620)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 configs/evb-ast2700_defconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configs/evb-ast2700_defconfig b/configs/evb-ast2700_defconfig
index c8fca7c57d..64be06df3e 100644
--- a/configs/evb-ast2700_defconfig
+++ b/configs/evb-ast2700_defconfig
@@ -31,9 +31,7 @@ CONFIG_SYS_MEMTEST_START=0x403000000
 CONFIG_SYS_MEMTEST_END=0x403001000
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_FIT=y
-CONFIG_FIT_SIGNATURE=y
 # CONFIG_BOOTMETH_EFILOADER is not set
-CONFIG_LEGACY_IMAGE_FORMAT=y
 CONFIG_USE_BOOTARGS=y
 CONFIG_BOOTARGS="console=ttyS12,115200n8 root=/dev/ram rw earlycon"
 CONFIG_USE_BOOTCOMMAND=y
-- 
2.34.1

